### DISCUSSION TITLE
How to add multiple characters as subscripts in formula?

### DISCUSSION THREAD

**Discussion by turbanisch** [typst/typst#2446] [General]
### Description

I would like to add multiple characters as subscripts in a math formula, similar to `\beta_{it}` in LaTeX. The typst documentation does not seem to point out how to mark the scope. Interpolating from the `sum` syntax, I was able to typeset `$ beta_(k=0) $` but not `$ beta_(it) $`. Using square or curly brackets did not work either. I assume this is already implemented though?

<img width="181" alt="image" src="https://github.com/typst/typst/assets/51969412/2f59eaea-d4e4-40d4-af76-e36eb44d150a">

### Operating system

Web app, macOS

### Typst version

- [X] I am using the latest version of Typst

**Comment by PgBiel** [comment]
Single letters are printed literally in math mode, while multiple letters are understood as variables or functions. In this case you need to write `beta_(i t)` instead of `beta_(it)`. Alternatively, you can write `beta_(italic("it"))`.
